documentation/src/main/paradox/runtime.md

```diff
@@ -45,4 +45,8 @@ Upon successful recovery, we log an info entry and run the effector, while we lo
 
 @@@ note { .tip title="Custom behavior" } 
 The built-in behavior is further customizable via a `customizeBehavior` function parameter that can be optionally passed into `deployEntity`. 
+@@@
+
+@@@ warning { title="Compatibility" }
+Since Akka [does not allow mixed versions](https://doc.akka.io/docs/akka/current/common/binary-compatibility-rules.html#mixed-versioning-is-not-allowed) in a project, Akka dependencies of `endless-runtime-akka` are marked a `Provided`. This means that your application `libraryDependencies` needs to directly include Akka as well.  
```

Comment:
As discussed, I suggest to document which version of Akka is required by endless. You can get it from your build and pass it to paradox, btw.

Reply:
Thanks @julienrf, yes excellent suggestion, I've pushed another commit